src/participant/prompts/query.ts

```diff
@@ -5,56 +5,62 @@ import { getHistoryMessages } from './history';
 import { getStringifiedSampleDocuments } from '../sampleDocuments';
 
 export class QueryPrompt {
-  static async getAssistantPrompt({
-    databaseName = 'mongodbVSCodeCopilotDB',
-    collectionName = 'test',
-    schema,
-    sampleDocuments,
-  }: {
-    databaseName?: string;
-    collectionName?: string;
-    schema?: string;
-    sampleDocuments?: Document[];
-  }): Promise<vscode.LanguageModelChatMessage> {
-    let prompt = `You are a MongoDB expert.
-
-Your task is to help the user craft MongoDB queries and aggregation pipelines that perform their task.
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user craft MongoDB shell syntax code to perform their task.
 Keep your response concise.
-You should suggest queries that are performant and correct.
-Respond with markdown, suggest code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
-You can imagine the schema.
+You must suggest code that is performant and correct.
+Respond with markdown, write code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
 Respond in MongoDB shell syntax using the \`\`\`javascript code block syntax.
-You can use only the following MongoDB Shell commands: use, aggregate, bulkWrite, countDocuments, findOneAndReplace,
-findOneAndUpdate, insert, insertMany, insertOne, remove, replaceOne, update, updateMany, updateOne.
 
 Concisely explain the code snippet you have generated.
 
 Example 1:
-use('');
-db.getCollection('').aggregate([
+User: Documents in the orders db, sales collection, where the date is in 2014 and group the total sales for each product.
+Response:
+\`\`\`javascript
+use('orders');
```

Comment:
I have noticed that you sometimes specify fake database and collection names and sometimes do not. Should we always either do or not do that? I initially provided examples with empty strings for namespaces to prevent the model from using these fake values instead real ones.

Reply:
I included the names here as they're in the user prompt example provided. I don't have strong evidence either way. I think the best change we're making for namespaces in this pr is removing the `You can imagine the schema.` line.

Happy to get rid of this change if you'd prefer it as it was before, I'm operating off of mostly hunches here since we don't have robust accuracy tests. Is there a better way we could guide the model here?

Reply:
got it! let's try this way 👍